contrail-kubernetes-master/hooks/contrail_kubernetes_master_hooks.py

```diff
@@ -90,8 +96,7 @@ def kube_api_endpoint_changed():
     changed |= _update_config(data, "kubernetes_api_secure_port", "port")
     config.save()
 
-    if is_leader():
-        changed |= utils.update_kubernetes_token()
+    changed |= utils.update_kubernetes_token()
```

Comment:
looks like this change and related change in utils file are not needed. or I miss something?